<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoManager</class>
 <widget class="QDialog" name="PhotoManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PhotoManager</string>
  </property>
  <property name="type" stdset="0">
   <string>standard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="pbCf">
     <property name="text">
      <string>CF</string>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="type" stdset="0">
      <string>standard</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="pbMs">
     <property name="text">
      <string>MS</string>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="type" stdset="0">
      <string>standard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="pbDvd">
     <property name="text">
      <string>DVD</string>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="type" stdset="0">
      <string>standard</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbCentral">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="pbSd">
     <property name="text">
      <string>SD</string>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="type" stdset="0">
      <string>standard</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>145</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="pbMm">
     <property name="text">
      <string>MM</string>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="type" stdset="0">
      <string>standard</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="pbFlash">
     <property name="text">
      <string>Flash</string>
     </property>
     <property name="iconSize">
      <size>
       <width>256</width>
       <height>256</height>
      </size>
     </property>
     <property name="type" stdset="0">
      <string>standard</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
